src/applications/personalization/profile360/components/PaymentInformationEditModalError.jsx

```diff
@@ -51,7 +57,7 @@ function UpdateAddressError({ closeModal }) {
   return (
     <p>
       We’re sorry. We couldn’t update your direct deposit bank information
-      because your address is missing or invalid. Please go back to{' '}
+      because your mailing address is missing or invalid. Please go back to{' '}
```

Comment:
The only address-related errors we are seeing are due to the _mailing_ address and not the residential address.